test/Interop/Cxx/deserialize-debug.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: %target-swift-frontend -enable-cxx-interop -I %S/Inputs %s -emit-ir -g | %FileCheck %s
```

Comment:
`test/Interop/Cxx/class/debug-info-irgen.swift`? Similarly, the header should be named `debug-info.h` to show the connection with the test.